packages/js/src/generators/library/library.ts

```diff
@@ -22,15 +24,20 @@ import {
   type ProjectNameAndRootOptions,
 } from '@nx/devkit/src/generators/project-name-and-root-utils';
 
-import {
-  addTsConfigPath,
-  getRelativePathToRootTsConfig,
-} from '../../utils/typescript/ts-config';
+import { requireNx } from '@nx/devkit/nx';
```

Comment:
We shouldn't need to use `requireNx` here. I'm not sure if its leftover from existing impl though. We use it inside devkit s.t. devkit is compatible w/ multiple major versions of Nx, but that's not a concern for individual plugins.

Reply:
updated 👍